src/SFA.DAS.FindApprenticeship.Jobs/Application/Handlers/RecruitIndexerJobHandler.cs

```diff
@@ -60,4 +70,45 @@ public async Task Handle()
             await azureSearchHelperService.UpdateAlias(Constants.AliasName, indexName);
         }
     }
+
+    public static IEnumerable<LiveVacancy> SplitLiveVacanciesToMultipleByLocation(IReadOnlyCollection<LiveVacancy> vacancies)
+    {
+        var multipleLocationVacancies = new List<LiveVacancy>();
+
+        foreach (var liveVacancy in vacancies)
+        {
+            var counter = 1;
+            foreach (var vacancyOtherAddress in liveVacancy.OtherAddresses)
+            {
+                var vacancy = DeepCopyJson(liveVacancy);
+                if (vacancy == null) continue;
+
+                vacancy.Id = $"{liveVacancy.Id}-{counter}";
+                vacancy.IsPrimaryLocation = false;
+                vacancy.OtherAddresses.RemoveAll(r =>
+                    r.AddressLine1 == vacancyOtherAddress.AddressLine1
+                    && r.AddressLine2 == vacancyOtherAddress.AddressLine2
+                    && r.AddressLine3 == vacancyOtherAddress.AddressLine3
+                    && r.AddressLine4 == vacancyOtherAddress.AddressLine4
+                    && r.Postcode == vacancyOtherAddress.Postcode
+                    && Math.Abs(r.Longitude - vacancyOtherAddress.Longitude) < Epsilon
+                    && Math.Abs(r.Latitude - vacancyOtherAddress.Latitude) < Epsilon);
+
+                if (liveVacancy.Address != null) vacancy.OtherAddresses.Add(liveVacancy.Address);
+                vacancy.Address = vacancyOtherAddress;
+                multipleLocationVacancies.Add(vacancy);
+                counter++;
+            }
+        }
+
+        return vacancies
+            .Concat(multipleLocationVacancies)
+            .ToList();
+    }
+
+    private static T? DeepCopyJson<T>(T input)
```

Comment:
I could have used normal mapping to clone the object but the Live vacancy has more than 50 Properties and some of them are complex objects as well, 

With this JSON serialisation, we could rely on and also if the properties get extended in the future, we don't need to amend this clone piece, it will be taken care.

This was the rational behind going with the JSON serialization.